src/server.ts

```diff
@@ -127,66 +133,46 @@ export const initServerFetcher =
 			});
 		}
 
-		// Skip persisted queries if operation is a mutation
+		// Skip automatic persisted queries if operation is a mutation
 		const queryType = getQueryType(query);
 		if (queryType === "mutation") {
-			return tracer.startActiveSpan(operationName, async (span) => {
+			return tracer.startActiveSpan(request.operationName, async (span) => {
 				try {
 					const response = await gqlPost(
 						url,
-						JSON.stringify({ documentId, operationName, query, variables }),
+						request,
 						{ cache, next },
-						options,
+						requestOptions,
 					);
 
 					span.end();
 					return response as GqlResponse<TResponse>;
-				} catch (err: any) {
+				} catch (err: unknown) {
 					span.setStatus({
 						code: SpanStatusCode.ERROR,
-						message: err?.message ?? String(err),
+						message: (err as Error).message ?? String(err),
```

Comment:
```suggestion
						message: err instanceof Error ? err.message : String(err),
```

The property access could fail thus breaking this